src/Http/Http.Extensions/gen/Microsoft.AspNetCore.Http.Generators.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+    <IsPackable>false</IsPackable>
+    <IsAnalyzersProject>true</IsAnalyzersProject>
+    <AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis.CSharp" PrivateAssets="all" />
+    <Reference Include="Microsoft.CodeAnalysis.Common" PrivateAssets="all" />
```

Comment:
Consider adding a reference to `Microsoft.CodeAnalysis.Analyzers`. It currently comes as a transitive package from `Microsoft.CodeAnalysis.Common`. The direct dependency will ensure https://github.com/dotnet/roslyn-analyzers/issues/6229 doesn't happen.